packages/utils/src/symbol.ts

```diff
@@ -0,0 +1,9 @@
+//
+// ES6 Symbols
+//
+
+export const useSymbols = typeof Symbol === 'function';
+
+export function createSymbolOrString(identifier: any) {
```

Comment:
I think identifier has to be a string

Reply:
Can we just use ES2015 Symbols directly and polyfill?

Reply:
Yes definitely.  That’s been on my todo forever